cmd/control/control.go

```diff
@@ -52,7 +54,7 @@ func NewCommand() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&logDir, "log-dir", "", "log directory")
 	cmd.PersistentFlags().StringVar(&endpoint, "endpoint", "localhost:8080", "server endpoint")
 	cmd.PersistentFlags().DurationVar(&dialTimeout, "dial-timeout", 10*time.Second, "server dial timeout")
-	cmd.PersistentFlags().DurationVar(&requestTimeout, "request-timeout", 3*time.Minute, "client request timeout")
+	cmd.PersistentFlags().DurationVar(&requestTimeout, "request-timeout", 300*time.Minute, "client request timeout")
```

Comment:
lets keep the default on 3, and let the user specify a special timeout. or... lets decide on 300 minute timeout if fuji

Reply:
k, I can add logic to increase it for fuji